RFC.md

```diff
@@ -129,8 +130,8 @@ NumPy scalars as 0-D arrays. This may cause small divergences in some cases like
 array([2, 4, 6])
 ```
 
-but we don't expect these to pose a big issue in practice. Note that in this
-implementation `torch_np.int32(2)` would return the same as `torch_np.asarray(2)`.
+but we don't expect these to pose a big issue in practice. Note that in the
+proposed implementation `np.int32(2)` would return the same as `np.asarray(2)`.
```

Comment:
in the proposed implementation --- when compiled through TorchDynamo via `@torch.compile`, correct?

Otherwise it make a naive reader in me utterly confused.